src/components/tag/Tag.types.ts

```diff
@@ -0,0 +1,7 @@
+type TLabelTagVariant = 'general' | 'info' | 'warning' | 'success';
+
+type TTagSizeToken = 25 | 50 | 75 | 100 | 200 | 300;
```

Comment:
to prevent errors I would suggest to import `TTextSizeToken` and do this: 

```typescript
export {
    TTextSizeToken as TTagSizeToken,
    ...
}
```